src/Controllers/TaxonomyDirectoryController.php

```diff
@@ -17,6 +17,9 @@
  * Class TaxonomyDirectoryController
  *
  * Controller for returning a list of pages tagged with a specific Taxonomy Term
+ *
+ * @template T
```

Comment:
Do we need `@template T` here? Seems like we're not using it

Reply:
Oops nope. Removed it.
I think I was originally adding it because this is a page controller - but then noticed that this controller isn't meant to be subclassed so didn't use it (and forgot to remove it)